client/packages.go

```diff
@@ -39,7 +39,7 @@ type Snap struct {
 	Status        string    `json:"status"`
 	Type          string    `json:"type"`
 	Version       string    `json:"version"`
-	Revision      int       `json:"revision"`
+	Revision      string    `json:"revision"`
```

Comment:
My original idea was actually to send the integer down the line to the client too, but I guess this makes more sense.